webapp/src/shared/components/ConfirmAlert/CustomConfirmAlert.tsx

```diff
@@ -0,0 +1,54 @@
+import { confirmAlert } from 'react-confirm-alert';
+import './CustomConfirmAlert.css';
+import { Button, ButtonProps, Text, TextVariant } from '../index';
+
+type CustomConfirmAlertProps = {
+  title: string;
+  message: string;
+  buttons: ButtonProps[];
+};
+
+export default function CustomConfirmAlert({
```

Comment:
This is technically not a component, since it's invoked like a function is called, doesn't return a ReactNode, but it's close enough, I think, and has UI and style, which we would expect from a regular component. Let me know if this bothers you!